portfolio/src/main/webapp/index.html

```diff
@@ -122,7 +122,8 @@ <h2>Learn about me in 30 seconds or less.</h2>
     </div>
     <br><br>
 
-    <!-- Div contains comment form(s). TODO - style the forms -->
+    <!-- Div contains comment form(s). -->
+    <!-- TODO - style the forms (https://github.com/austinteshuba/GoogleSTEP2020/issues/24) -->
```

Comment:
Out of curiosity: The way we do this internally is
`TODO(<issue link>): <issue title | "localized" issue title>`
(It looks alright internally because our issue links are `b/<issue id>`)

Reply:
Oh interesting - What is the localized Issue title? 

For example, what would the localized issue title be in this case?

Reply:
This style seems fine to me, given how long the issue URLs are. If you find a short link notation for issues that generates links in code on GitHub, let me know!